docs/en/contribution/release-agent.md

```diff
@@ -0,0 +1,171 @@
+
+## Test your settings and package
+
+```shell
+## Make sure local compiling passed
+> cargo build
+
+## Create package.xml from package.xml.tpl
+> cargo run -p scripts --release -- create-package-xml --version x.y.z --notes "`cat /tmp/notes.txt`"
+
+## Create local package. The skywalking_agent-x.y.z.tgz should be found in project root
+> pecl package
+```
+
+## Sign the package
+
+Tag the commit ID of this release as v`x.y.z`.
+
+After set the version in `Cargo.toml` with the release number, package locally. Then run the following commands to sign
+your package.
+
+```shell
+> export RELEASE_VERSION=x.y.z
+
+## The package should be signed by your Apache committer mail.
+> gpg --armor --detach-sig skywalking_agent-{$RELEASE_VERSION}.tgz
+
+> shasum -a 512 skywalking_agent-{$RELEASE_VERSION}.tgz > skywalking_agent-{$RELEASE_VERSION}.tgz.sha512
+```
+
+After these, the source tar with its signed asc and sha512 are ready.
+
+## Upload to Apache SVN and tag a release
+
+1. Use your Apache ID to log in to `https://dist.apache.org/repos/dist/dev/skywalking/php`.
+2. Create a folder and name it by the release version and round, such as: `x.y.z`
+3. Upload tar ball, asc, sha512 files to the new folder.
+
+## Call a vote in dev
+
+Call a vote in `dev@skywalking.apache.org`
+
+```
+Mail title: [VOTE] Release Apache SkyWalking version PHP x.y.z
+
+Mail content:
+Hi All,
+This is a call for vote to release Apache SkyWalking PHP version x.y.z.
+
+Release Candidate:
+
+* https://dist.apache.org/repos/dist/dev/skywalking/php/x.y.z/
+* sha512 checksums
+- xxxxxxxx skywalking_agent-x.y.z.tgz
+
+Release Tag :
+
+* (Git Tag) vx.y.z
+
+Release CommitID :
+
+* https://github.com/apache/skywalking-php/tree/{commit-id}
+
+Keys to verify the Release Candidate :
+
+* https://dist.apache.org/repos/dist/release/skywalking/KEYS
+
+Guide to build the release from source :
+
+* https://github.com/jmjoy/skywalking-php/blob/master/docs/en/contribution/compiling.md
+
+Voting will start now (Date) and will remain open for at least 72
+hours, Request all PMC members to give their vote.
+[ ] +1 Release this package.
+[ ] +0 No opinion.
+[ ] -1 Do not release this package because....
+```
+
+## Vote Check
+
+The voting process is as follows:
+
+1. All PMC member votes are +1 binding, and all other votes are +1 but non-binding.
+1. If you obtain at least 3 (+1 binding) votes with more +1 than -1 votes within 72 hours, the release will be approved.
+
+## Publish the release
+
+1. Move source codes tar and distribution packages to `https://dist.apache.org/repos/dist/release/skywalking/`.
+
+   ```shell
+   > export SVN_EDITOR=vim
+   > svn mv https://dist.apache.org/repos/dist/dev/skywalking/php/x.y.z https://dist.apache.org/repos/dist/release/skywalking/php
+   ....
+   enter your apache password
+   ....
+   ```
+
+2. Pecl publish package on [skywalking_agent](https://pecl.php.net/package/skywalking_agent).
```

Comment:
Is this package registered by you? If so, is this management platform able to add others as owner/admin in case we have more release managers in the future?

Reply:
This package has not been released. I have no PECL publishing experience at present. @heyanlong  should know the process.

Reply:
My only concern is, who could release on PECL. The docs of the release process should be doable for all PMC members(although in reality, only you two would do)

Reply:
I will request the pecl account now, via <https://pecl.php.net/account-request.php>.

Reply:
OK, I will update the doc about publishing to PECL.